src/java/org/apache/cassandra/service/accord/serializers/CommandsForKeySerializer.java

```diff
@@ -350,6 +354,8 @@ public static ByteBuffer toBytesWithoutKey(CommandsForKey cfk)
 
             VIntCoding.writeUnsignedVInt(prevEpoch, out);
             VIntCoding.writeUnsignedVInt(prevHlc, out);
+            VIntCoding.writeUnsignedVInt32(cfk.redundantBefore().flags(), out);
```

Comment:
Why not serialize the entire `redundantBefore` `txnId` since `prevEpoch` and `prevHlc` are all part of it?

Reply:
Hmm, looks like the standard serialiser takes up more space. But sure, don't mind using it - but perhaps we should encode epoch/hlc/flags separately in the standard serialiser.